hazelcast/src/main/java/com/hazelcast/sql/impl/schema/Mapping.java

```diff
@@ -127,4 +127,59 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(name, externalName, type, mappingFields, options);
     }
+
+    public String unparse() {
+        StringBuffer buffer = new StringBuffer();
+
+        buffer.append("CREATE MAPPING");
```

Comment:
this code is similar to `SqlCreateMapping.unparse`. can it be reused?

Reply:
so maybe we can to the other way around and use this method in static `SqlCreateMapping.unparse`? Currently we have 3 copies of almost the same code. SQL module implementation uses Calcite SQL generator which is a bit richer than plain string builder, but if we do not want to introduce Calcite dependency here we can use stringbuilder.

Reply:
The queries to unparse are quite simple, and different. 
`SqlCreateMapping.unparse` serves for other places in `hazelcast-sql`, for me it was quite important to have same method to call in same entities, like `Mapping` etc.

Reply:
we can remove static `SqlCreateMapping.unparse` and use `Mapping.unparse` instead. Static `SqlCreateMapping.unparse` is used only in error generation and `SqlMappingDdlTask`